tests/ui/associated-types/issue-20005.stderr

```diff
@@ -4,11 +4,11 @@ error[E0277]: the size for values of type `Self` cannot be known at compilation
 LL |     ) -> <Dst as From<Self>>::Result where Dst: From<Self> {
    |                                                 ^^^^^^^^^^ doesn't have a size known at compile-time
    |
-note: required by a bound in `From`
+note: required by an implicit `Sized` bound in `From`
   --> $DIR/issue-20005.rs:1:12
    |
 LL | trait From<Src> {
-   |            ^^^ required by this bound in `From`
+   |            ^^^ required by the implicit `Sized` requirement on this bound in `From`
```

Comment:
Could you rephrase this slightly? “requirement on this bound” isn't correct since `Src` isn't a bound but a generic parameter and a bound can't “have” a requirement (it *is* a requirement) and there can't be anything “on” a bound. Of course, this is preexisting but now it's slightly worse. I would go with something like this maybe:

```suggestion
   |            ^^^ required by the implicit `Sized` requirement on this type parameter in `From`
```

Reply:
Done.